include/planner/ODG_PF.hpp

```diff
@@ -0,0 +1,172 @@
+// int argMax = std::distance(x.begin(), std::max_element(x.begin(), x.end()));
+# pragma once
+
+#include <math.h>
+#include <vector>
+#include <iostream>
+// #include <matplotlibcpp.h>
+using namespace std;
```

Comment:
remove all _using namespace xyz_  statements, and encapsulate code in namespace "huro".